WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -13,6 +13,8 @@ import CoreTelephony
 
     private var userName: String?
     private var userEmail: String?
+    private var needToRegisterDevice = false
```

Comment:
Do we need both properties here? Could we just check whether we have a deviceID, and if so use it to register the device?

Reply:
Oh. Hehe. Good point.